src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -193,4 +200,19 @@ public String getAdditionalSourceFileLocation() {
     List<String> getAdditionalHeaderFiles() {
         return androidAdditionalHeaderFiles;
     }
+
+   /*
+    * Copies the .cap files from the jar resource and store them in
+    * a directory. Return that directory
+    */
+    private String getCapCacheDir() throws IOException {
```

Comment:
IMO is it better to return a `Path`. It denotes what we are returning and allows for immediate usage of Path APIs from then on.